### ISSUE TITLE
Publish Microsoft.SymbolUploader.Build.Task to dotnet-core feed

### ISSUE THREAD

**Issue by tmat** [dotnet/arcade#3037]
This is required before we can switch to NuGet.config.

**Comment by tmat** [comment]
@jcagme @markwilkie @mikem8361

**Comment by mikem8361** [comment]
This repo needs to switch to the latest Arcade (issue https://github.com/dotnet/core-eng/issues/6042) and switch to dnceng (issue https://github.com/dotnet/core-eng/issues/6043).

The diagnostics team has a lot of engineering debt.

**Comment by markwilkie** [comment]
Thanks @mikem8361 .  We're hoping to make the switch (in Arcade first) by 6/26, starting to test Wed the 19th.  Is this unreasonable?  If so, let's chat to see what our options are.

**Comment by mikem8361** [comment]
I'm not sure I have the bandwidth right now to do this. We have a lot of Preview 7 work for both the runtime and our tools.  

@tommcdon

**Comment by tommcdon** [comment]
@markwilkie we should be able to get to this after the coreclr preview 7 snap

**Comment by markwilkie** [comment]
Makes sense and sounds reasonable.  @jcagme - what are the ramifications here?  Perhaps we can get Arcade up and going, but leave the symbol uploader until post p7?

**Comment by tmat** [comment]
We just manually pushed the current package to the feed so as long as we do not need updates we can do this work later.

**Comment by tmat** [comment]
I have updated the description since we don't need to consider this as "required" for nuget.config switch atm.

**Comment by JohnTortugo** [comment]
I think it's relevant to link these issues here: https://github.com/dotnet/core-eng/issues/6042 https://github.com/dotnet/core-eng/issues/6043

**Comment by mmitche** [comment]
I've been pushing these to dotnet-tools.  Should be good to go. I think this issue can be closed.

**Comment by mikem8361** [comment]
I’m using this issue to track doing the engineering work to the DotNet-SymbolUploader repo so I rather you leave it option.  I’ll will get to it soon.
